lib/web_game.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'datastore'
+require 'tic_tac_toe'
+
+class WebGame
+    def initialize(datastore=Datastore.new, game=TicTacToe::Game.new, turn=TicTacToe::Turn.new)
+        @datastore = datastore
+        @game = game
+        @turn = turn
+    end
+
+    def load_state(game_id)
+        @game.state = @datastore.load(game_id, :state)
+    end
+
+    def load_turn(game_id)
+        @turn.set_turn(@datastore.load(game_id, :turn))
+    end
+
+    def load_all()
+        @datastore.load_all()
+    end
+
+    def clear_all()
+        @datastore.clear
+    end
+
+    def save_game(game_id)
+        data = { game_id=> {'state': @game.state, 'turn': @turn.get_turn} }
+        store(data)
+    end
+
+    def start_game(game_id)
+        data = { game_id=> { "state"=> ['','','','','','','','',''], "turn"=> "X"}  }
```

Comment:
Would `@game.state` give you the same empty board array  `['','','','','','','','','']`?

Reply:
If so, it may be better to hide the array behind `@game` object.